inngest/client/client.go

```diff
@@ -23,6 +24,8 @@ type Client interface {
 
 	AllEvents(ctx context.Context, query *EventQuery) ([]Event, error)
 	Events(ctx context.Context, query *EventQuery, cursor *Cursor) (*PaginatedEvents, error)
+	RecentEvent(ctx context.Context, workspaceID uuid.UUID, eventID ulid.ULID) (*ArchivedEvent, error)
+	RecentEvents(ctx context.Context, workspaceID uuid.UUID, eventName string, count int64) ([]ArchivedEvent, error)
```

Comment:
Is that nomenclature okay? To fit with the ideal these would be `Event` and `Events`, but that'd require renaming `Events` to `EventTriggers`; I didn't want to be too destructive. ;)

Reply:
This is totally okay.